Casks/g/getoutline.rb

```diff
@@ -0,0 +1,33 @@
+cask "getoutline" do
+  arch arm: "arm64", intel: "x64"
+
+  version "1.3.1"
+  sha256 :no_check
+
+  # arm64 (apple-silicon)
+  on_arm do
+    url "https://desktop.getoutline.com/mac/dmg/arm64"
+  end
+  # x64 (intel)
+  on_intel do
+    url "https://desktop.getoutline.com/mac/dmg/x64"
+  end
+
```

Comment:
You can merge those blocks into one,
```suggestion
    url "https://desktop.getoutline.com/mac/dmg/#{arch}"
```